Core/HLE/sceKernelThread.cpp

```diff
@@ -3095,6 +3100,13 @@ void __KernelExecuteMipsCallOnCurrentThread(u32 callId, bool reschedAfter)
 	call->savedV1 = currentMIPS->r[MIPS_REG_V1];
 	call->savedId = cur->currentMipscallId;
 	call->reschedAfter = reschedAfter;
+	
+	// Also need to backup other regs which might be used by the called mips function, needed to prevent corrupting the original regs (ie. Dissidia 012 with more than 2 players, Lord of Arcana)
```

Comment:
I don't like these changes.

First off, why is the callback being triggered at times when this would be a concern?  It's very unlikely this would be happening on the PSP.

If it does, you I think this should be handled inside `AfterMatchingMipsCall` instead of here, since I do not think this is the correct behavior for other mipscalls.

-[Unknown]